daisy/compute/compute.go

```diff
@@ -241,7 +245,17 @@ func shouldRetryWithWait(tripper http.RoundTripper, err error, multiplier int) b
 
 // NewClient creates a new Google Cloud Compute client.
 func NewClient(ctx context.Context, opts ...option.ClientOption) (Client, error) {
-	o := []option.ClientOption{option.WithScopes(compute.ComputeScope, compute.DevstorageReadWriteScope)}
+	// Set these scopes to be align with compute.NewService
+	o := []option.ClientOption{
+		option.WithScopes(
+			compute.CloudPlatformScope,
+			compute.ComputeScope,
+			compute.ComputeReadonlyScope,
+			compute.DevstorageFullControlScope,
+			compute.DevstorageReadOnlyScope,
+			compute.DevstorageReadWriteScope,
```

Comment:
I'm a bit of a noob in terms of scopes; if the account executing Daisy doesn't have a permission enabled (say: DevstorageFullControlScope), would there be a failure due to adding these new scopes?

In other words, is there a possibility that pushing this to Daisy will cause new breakages?

Reply:
No. They are not related at all. This scope is just about which kind of operations can happen, it's a self-limitation from client side.